src/components/Reports/ReportEdit.tsx

```diff
@@ -212,23 +219,25 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
         (!nursery.buildStartedDate ||
           !nursery.buildCompletedDate ||
           !nursery.operationStartedDate ||
-          !nursery.capacity ||
+          nursery.capacity === null ||
           !buildStartedDateValid(nursery) ||
           !buildCompletedDateValid(nursery) ||
-          !operationStartedDateValid(nursery))
+          !operationStartedDateValid(nursery) ||
+          emptyWorkerField(nursery))
       );
     });
     const emptyPlantingSitesFields = iReport.plantingSites?.some((plantingSite) => {
       const speciesDataMissing =
-        plantingSite.species?.some((sp) => !sp.totalPlanted || !sp.mortalityRateInField) ?? false;
+        plantingSite.species?.some((sp) => sp.totalPlanted === null || sp.mortalityRateInField === null) ?? false;
       return (
         plantingSite.selected &&
-        (!plantingSite.totalPlantingSiteArea ||
-          !plantingSite.totalPlantedArea ||
-          !plantingSite.totalTreesPlanted ||
-          !plantingSite.totalPlantsPlanted ||
-          !plantingSite.mortalityRate ||
-          speciesDataMissing)
+        (plantingSite.totalPlantingSiteArea === null ||
```

Comment:
Check for null instead of falsey. Zero was getting interpreted as an invalid value